package.json

```diff
@@ -31,14 +31,13 @@
   ],
   "devDependencies": {
     "eslint": "^7.31.0",
-    "go-ipfs": "^0.9.1",
-    "ipfs": "^0.56.0",
-    "ipfs-http-client": "^51.0.0",
-    "ipfsd-ctl": "^9.0.0",
+    "go-ipfs": "^0.10.0-rc1",
+    "ipfs": "^0.58.2",
+    "ipfsd-ctl": "^10.0.3",
     "mocha": "^9.0.3",
     "node-pre-gyp": "^0.17.0",
     "nyc": "^15.1.0",
-    "orbit-db-test-utils": "^1.2.0",
+    "orbit-db-test-utils": "github:tabcat/orbit-db-test-utils#upgrade-ipfs",
```

Comment:
Can't have this in the canonical `main`, probably just some housekeeping you gotta do.

Reply:
good point :+1: we should not forget that before we merge and/or publish.
opinion on not publishing anything until `orbit-db` is working on the upgrade-ipfs forks?

Reply:
nah, it's ok if things go to `@next` that are incompatible. I don't want to delay merging/publishing a second longer than we have to